libs/beacon/include/beacon/beacon_manager.hpp

```diff
@@ -17,36 +17,29 @@
 //
 //------------------------------------------------------------------------------
 
-#include "crypto/bls_base.hpp"
-#include "crypto/bls_dkg.hpp"
-#include "crypto/ecdsa.hpp"
-#include "crypto/identity.hpp"
-
-#include <unordered_map>
+#include "crypto/mcl_dkg.hpp"
+#include "dkg/dkg_messages.hpp"
 
 namespace fetch {
 namespace dkg {
 
 class BeaconManager
 {
 public:
```

Comment:
Might need some high-level comments in this file (I can do this)

Reply:
Thanks.